ledger-service/http-json-testing/src/main/scala/com/daml/http/HttpServiceTestFixture.scala

```diff
@@ -353,6 +353,14 @@ object HttpServiceTestFixture extends LazyLogging with Assertions with Inside {
       .fold(e => throw new IllegalArgumentException(s"cannot sign a JWT: ${e.shows}"), identity)
   }
 
+  def jwtForParties(
+      actAs: List[String],
+      readAs: List[String],
+      ledgerId: String,
+      withoutNamespace: Boolean = false,
+      admin: Boolean = false,
+  ): Jwt = jwtForParties(actAs, readAs, Some(ledgerId), withoutNamespace, admin)
```

Comment:
adressed with be8281fc6f346e4db7ea919a93d25e8b94be4bff too. It could be propagated further but that would result in even more changes where I don't see much value in right now.